tests/backup/backup_helper.go

```diff
@@ -5996,3 +6004,59 @@ func CreateKubevirtBackupRuleForAllVMsInNamespace(ctx context.Context, namespace
 
 	return ruleStatus, ruleName, nil
 }
+
+func validateCRCleanup(resourceName string, clusterName string, orgID string,
```

Comment:
I would suggest having 2 different functions called `ValidateRestoreCRCleanup` and `ValidateBackupCRCleanup` which will take only restore/backup object

If you want a common function you can create a function called `ValidateCRCleanup` which will take an interface. If the interface is backup object the you will call `ValidateBackupCRCleanup` else you will call `ValidateRestoreCRCleanup`

Reply:
Created ValidateRestoreCRCleanup and ValidateBackupCRCleanup wrapper methods with restore and backup object as input.